.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+
+
+
+## Code Review Checklist
+ - [ ] Does the PR describe what changes are being made?
+ - [ ] Does the PR describe why the changes are being made?
+ - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
+ - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
+ - [ ] Could the PR be broken into smaller PRs?
+ - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
+ - [ ] Does the new code have enough documentation? (comments and or diagrams)
```

Comment:
"enough documentation" seems a little vague. I'm personally curious, where do we expect people to be adding documentation? Seems like there are different places to do so. For example there are two different wikis: https://wiki.storj.io/ and https://github.com/storj/storj/wiki (which apparently is sourced from the storj/docs repo).

Reply:
good question! i think in this case we mean https://godoc.org/storj.io/storj and https://github.com/storj/docs, yeah. i'll add links

Reply:
^that first link doesn't work for me JT. but yeah agree, maybe the checklist should say "Does the new code have up-to-date documentation that answers 'how do I use it?' and 'what does it do?'"?

Reply:
ugh, our meta tags are still broken on storj.io so godoc doesn't work. i'll pull the link until i fix that